extensions/wikia/ARecoveryEngine/templates/ARecoveryEngineApiController_getPageFairBootstrapBottomBody.mustache

```diff
@@ -13,6 +14,13 @@
 			adonis.conditionalAdRendering = true;
 			adonis.conditionalIframeRendering = true;
 			adonis.transport = 'https://9k6y.b2rrns9dgbx56tf5kc.agency/http';
+			adonis.od = function(detectionStatus) {
+				win.ads.runtime = win.ads.runtime || {};
+				win.ads.runtime.pf = win.ads.runtime.pf || {};
+				win.ads.runtime.pf.recovering = detectionStatus;
+				// the ATF slots are already defined so we need to update their targeting
```

Comment:
yes, because the PF detects blocking basing on ATF requests that are sent.
Even if they are not updated, updating their targeting won't make any harm :)